src/runner/general_settings.h

```diff
@@ -22,4 +22,4 @@ struct GeneralSettings
 json::JsonObject load_general_settings();
 GeneralSettings get_general_settings();
 void apply_general_settings(const json::JsonObject& general_configs, bool save = true);
-void start_initial_powertoys();
+void start_initial_powertoys();
```

Comment:
Add a newline at the end of the file. Same for all .cpp and .h files with this symbol.

Reply:
Is it a requirement? Code formating `powershell src\codeAnalysis\format_sources.ps1` does not add empty line at the end of the file if it is not present there. So if it is a requirement we should configure it there.

Reply:
[Git likes it that way](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)

Reply:
done

Reply:
So was the new line added or not? Because it looks like it wasn't.